### PR TITLE
Bump to v5.0.0-RC7

### PR DESCRIPTION
**Pull request by mheon**
As the title says

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: version/rawversion/version.go [hunk 1]
```diff
@@ -7,4 +7,4 @@ package rawversion
 //
 // NOTE: remember to bump the version at the top of the top-level README.md
 // file when this is bumped.
-const RawVersion = "5.0.0-rc7"
+const RawVersion = "5.0.0-dev"
```
Comment (vrothberg) [inline version/rawversion/version.go:10]: Should this be 5.1.0-dev?

**Review by vrothberg — COMMENTED** [review]
LGTM, restarted flaked jobs

File: version/rawversion/version.go (hunk 1 above)
Comment (lsm5) [reply version/rawversion/version.go:10]: nope, this is already on v5.0 branch

Comment (lsm5) [reply version/rawversion/version.go:10]: but ya, we need to bump main to 5.1.0-dev

Comment (lsm5) [reply version/rawversion/version.go:10]: https://github.com/containers/podman/pull/22054

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL and merge

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -83,10 +83,12 @@
 - A new API endpoint, `/libpod/images/$name/resolve`, has been added to resolve a (potential) short name to a list of fully-qualified image references Podman which could be used to pull the image.
 - Fixed a bug where the List API for Images did not properly handle filters and would discard all but the last listed filter.
 - Fixed a bug in the Docker Create API for Containers where entries from `/etc/hosts` were copied into create containers, resulting in incompatibility with network aliases.
+- Fixed a bug in the Libpod and Docker Exec APIs for Containers which caused incorrect header values to be set when upgrading a connection for an interactive exec session.
 - The API bindings have been refactored to reduce code size, leading to smaller binaries ([#17167](https://github.com/containers/podman/issues/17167)).
 
 ### Misc
 - Failed image pulls will now generate an event including the error.
+- The gzip compression library used for sending build contexts, improving performance for remote `podman build`.
```
Comment (Luap99) [inline RELEASE_NOTES.md:91]: should this be `Changed the ...`? It sounds odd without that, anyhow I am fine to merge to not wait for another CI cycle and fic that up for the final release.

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

File: RELEASE_NOTES.md (hunk 2 above)
Comment (mheon) [reply RELEASE_NOTES.md:91]: Yeah, that sounds better

**Pull request merged**